functions/gravityForms/yupSchema/StringSchemaFactory.js

```diff
@@ -0,0 +1,71 @@
+import * as Yup from 'yup'
+
+/**
+ * Create Yup string schema object.
+ *
+ * @author WebDevStudios
+ * @see https://github.com/jquense/yup#string
+ * @returns {Object} Yup validationSchema Object.
+ */
+export default class StringSchemaFactory {
+  /**
+   * Constructor.
+   *
+   * Note: wp-graphql-gravity-forms plugin may need to be replaced.
+   *
+   * @author WebDevStudios
```

Comment:
I'm very much OK with omitting `@author` once we're inside a class. You've already defined the `@author` on the class itself. This just becomes "noise" at this point.

Reply:
I'm on board with this idea. I'll get it taken care of.